requirements.txt

```diff
@@ -1,6 +1,6 @@
 attrs
 jax==0.2.8
-jaxlib>=0.1.59
+jaxlib==0.1.59
```

Comment:
jaxlib has made several changes in the IR, this makes sure the CI passes.

Reply:
Maybe leave a issue, we may want to revisit this.

Reply:
done